incidents/src/main/java/fr/openent/incidents/export/IncidentsCSVExport.java

```diff
@@ -62,7 +62,7 @@ private String getProtagonists(JsonArray protagonists) {
                     .append(" ")
                     .append(protagonist.getJsonObject("type").getString("label"))
                     .append(": ")
-                    .append(protagonist.getJsonObject("student").getString("displayName"))
+                    .append(protagonist.getJsonObject("student") != null ? protagonist.getJsonObject("student").getString("displayName") : "")
```

Comment:
Tu peux gérer avec les valeurs par défaut :

```suggestion
 .append(protagonist.getJsonObject("student", new JsonObject()).getString("displayName", ""))
```

Si jamais ton student est null => tu envoies un JsonObject "vide" et au final comme ton displayName vaudra "null" => tu obtiens une chaine vide